src/main/java/org/folio/entlinks/domain/repository/AuthorityDataRepository.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.entlinks.domain.repository;
+
+import java.util.Collection;
+import java.util.UUID;
+import org.folio.entlinks.domain.entity.AuthorityData;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface AuthorityDataRepository extends JpaRepository<AuthorityData, UUID> {
+
+  @Modifying
+  @Query("update AuthorityData a set a.naturalId = :naturalId where a.id = :id")
+  void updateNaturalIdById(@Param("naturalId") String naturalId, @Param("id") UUID id);
+
+  @Modifying
+  @Query("update AuthorityData a set a.deleted = true where a.id in :ids")
```

Comment:
Shouldn't :ids be in brackets? like `(:ids)`

Reply:
No, it's not required. Spring JPA do it by itself